### ISSUE TITLE
The StreamField doesn't work for django admin inlines

### ISSUE THREAD

**Issue by holasoftware** [raagin/django-streamfield#45]
The StreamField doesn't work in django admin inlines because it's not initializating when a new inline form is added to the inline  formset. Maybe the trick is to listen to this event `formset:added` and then to do the initialization.

Maybe adding something like this to  `streamfield_widget.js` can work:

````
document.addEventListener("formset:added", function(event) {
    let app_node = event.target.querySelector('.streamfield_app');

    let app = createApp(App, {app_node}).mount(app_node.querySelector('.mount-node'));
    window.streamapps[app_node.id] = app;
});
`

**Comment by raagin** [comment]
Yes, it works! I will add it in the next release. Thank you.

**Comment by holasoftware** [comment]
Many thanks for the new release. Another issue is that the translation texts are included every time that the streamfield is used and also the stream_text global variable is overridden every time. As a consequence and another issue at the same time, the attribute stream_texts.help_text has the value related to the last stream field widget and all the stream field widgets in the html page has that last value.

My idea has been to generate a javascript dynamically for the stream texts and to add that javascript as a media dependency for the stream field widget. The help text is specific for every context and I add that value as an attribute of the vue app element.

This is my pull request:
https://github.com/raagin/django-streamfield/pull/46

**Comment by raagin** [comment]
Great, I've merged this into the 2.3.0 release.
